framework/src/components/cache/cache.js

```diff
@@ -43,9 +43,12 @@ class Cache {
 		// but cache server is not available.
 		return new Promise(resolve => {
 			this.client = redis.createClient(this.options);
-			this.client.once('error', err => {
+			this.client.once('error', error => {
```

Comment:
I see we changed `err` to `error` here, and at line 77 we keep the same `err` variable? Did you try to make this consistent? If yes, we can decide upon `err` or `error` usage.

Reply:
I see you used `err` almost everywhere, maybe change here as well to `err`